### ISSUE TITLE
Issues with 1.3

### ISSUE THREAD

**Issue by bonacci31** [Avangelista/CarTube#20]
Lot of issues in cars with no touch screen and scroll navigation. 

1. 1st video after connection starts muted and can not be unmuted.
2. Every video after the first one just doesn't start
3. Scroll navigation still doesn't work. 
4. "Play on CarPlay" share extensions breaks the app

Rolling back to 1.2...

**Comment by Avangelista** [comment]
Thanks for the detailed feedback. I’ll see what I can do about a limited feature app dedicated to those of you without a touch screen in the near future.
Regarding the share extension, what’s the behaviour you’ve observed?

**Comment by UTfan2023** [comment]
Thank you for creating the IPA - the app is useful and works well for me. 

The only question I have is related to purchased movies. I am still having issues playing a movie that I have downloaded to my iPhone. The error I receive with this version 1.3 is "Your browser can't play this video" which, based on limited research, may be a mobile Youtube player issue rather than an issue specifically tied to CarTube. Do you know if this is something that could be fixed in the future? 

Thanks!

**Comment by Avangelista** [comment]
> Thank you for creating the IPA - the app is useful and works well for me.
> 
> The only question I have is related to purchased movies. I am still having issues playing a movie that I have downloaded to my iPhone. The error I receive with this version 1.3 is "Your browser can't play this video" which, based on limited research, may be a mobile Youtube player issue rather than an issue specifically tied to CarTube. Do you know if this is something that could be fixed in the future?
> 
> Thanks!

This is due to the movie having DRM. unfortunately, the CarPlay display is unable to play any DRM content.

**Comment by bonacci31** [comment]
> Thanks for the detailed feedback. I’ll see what I can do about a limited feature app dedicated to those of you without a touch screen in the near future. Regarding the share extension, what’s the behaviour you’ve observed?

Sorry for by quick assumption on the share button. It didn't work on a sideloaded YouTube app. It works on the original app but the video doesn't start as I mentioned in the 2nd point.

**Comment by UTfan2023** [comment]
Ah ok. I should have connected the dots on that - thanks for the quick reply!

**Comment by augustobob** [comment]
Hey Avangelista, thank you for your work and support, I’d like to suggest you to look into the iOS app vinegar which enables YouTube videos in safari to play with the default player, maybe it would make the app easier to select video quality and play videos more reliable.

**Comment by diegogarciaarriaza** [comment]
Yes. Same behaivor reported yet in my no touch screen system
1. Video start muted
2. Video is not played on fullscreen mode

Share extension works well

Thanks for your dedication.

**Comment by diegogarciaarriaza** [comment]
Do you have any news about this fixes?
Or even any way of help you?

Thanks!

**Comment by severynhaiduk** [comment]
> Lot of issues in cars with no touch screen and scroll navigation.
> 
> 1. 1st video after connection starts muted and can not be unmuted.
> 2. Every video after the first one just doesn't start
> 3. Scroll navigation still doesn't work.
> 4. "Play on CarPlay" share extensions breaks the app
> 
> Rolling back to 1.02...

Does that old version supports notouchscreen cars better? No mute?

**Comment by bonacci31** [comment]
> > Lot of issues in cars with no touch screen and scroll navigation.
> > 
> > 1. 1st video after connection starts muted and can not be unmuted.
> > 2. Every video after the first one just doesn't start
> > 3. Scroll navigation still doesn't work.
> > 4. "Play on CarPlay" share extensions breaks the app
> > 
> > Rolling back to 1.02...
> 
> Does that old version supports notouchscreen cars better? No mute?

Yes in 1.02 it can play videos without mute but still not controls. You can only play the video url via the app
